src/cc2olx/models.py

```diff
@@ -90,19 +76,6 @@ def __repr__(self):
         )
         return text
 
-    def serialize(self):
```

Comment:
Ah yeah, I think this was my first attempt at serialization, before we deferred to the single-XML file approach.